src/components/Tooltip/Tooltip.mdx

```diff
@@ -96,6 +96,19 @@ on a disabled `Button`, pass the `shouldWrapChildren` prop.
 
 <Canvas of={TooltipStories.DisablingTooltipButton} />
 
+### Placement
+
+The `placement` prop adjusts where the tooltip is displayed relative to its children.
+
+<Canvas of={TooltipStories.TooltipPlacement} />
+
+### Offset
+
+The `offset` prop sets the distance along the main and cross axis between the tooltip and its reference element.
+`offset` is just static pixel values, so be careful to check where your tooltip displays on all breakpoints.
```

Comment:
Give a visual example of the prop value in " pixel values, such as `[100, 100]`, so be careful..."  or somewhere else.